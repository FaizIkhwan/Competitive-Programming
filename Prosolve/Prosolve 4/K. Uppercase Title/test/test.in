5
a b c
aa bbb cccc
linus torvalds
welcome to prosolve 4.0!!!
muhammad ibn musa al Khwarizmi