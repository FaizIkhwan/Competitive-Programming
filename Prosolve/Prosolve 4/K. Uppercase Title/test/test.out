A B C
Aa Bbb Cccc
Linus Torvalds
Welcome To Prosolve 4.0!!!
Muhammad Ibn Musa Al Khwarizmi
